src/main/resources/default_config.json

```diff
@@ -14,7 +14,7 @@
     "publish_interval": 10
   },
   "server": {
-    "port": 8080,
+    "port": 4101,
```

Comment:
I see the port is configured back to 8080 as part of this child PR - https://github.com/hwslabs/iam/pull/127/files#diff-5b37962640da70b526f471b12391ae9b865d5719ba4cc26e1f7d9efc6d6e21f3

Please ensure that the port number is not changed when pushing to main.